### DISCUSSION TITLE
libvips, libwebp, and CVE-2023-4863

### DISCUSSION THREAD

**Discussion by ENOTTY** [libvips/libvips#3675] [General]
Hi all,

I'm wondering which version of libvips bundled a libwebp version at 1.3.1 or earlier.

Based on the release dates, I wager that libvips 8.14.5 likely built with libwebp 1.3.2, but getting some confirmation of that would be useful. libvips 8.14.4 predates libwebp 1.3.2 so I wager it does use an older version of libwebp.

I'm interested in this question because libwebp 1.3.2 (released Sept 14 or so) fixed CVE-2023-4863. For more information about CVE-2023-4863, see https://blog.isosceles.com/the-webp-0day/

Thanks!

**Comment by lovell** [comment]
The only prebuilt binaries that libvips provides are for Windows. If you use these then please upgrade to the 8.14.5 release, which includes libwebp 1.3.2 - please see https://github.com/libvips/build-win64-mxe/releases/tag/v8.14.5

**Comment by ENOTTY** [comment]
Thank you for the timely information!
